.changeset/olive-spoons-return.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hyperlane-xyz/registry': patch
```

Comment:
major because of warp name changes?

Reply:
🤷‍♂️ up to you, I'm personally happy with a minor but I'm the resident semver ignorer